java/test/jmri/jmrit/display/layoutEditor/LayoutEditorTest.java

```diff
@@ -24,6 +24,37 @@
 
     @Rule
     public RetryRule retryRule = new RetryRule(2); // allow 2 retries
+    
+    @BeforeClass
+    static public void setUpClass() {
+          JUnitUtil.setUp();
```

Comment:
You need to remove this setUpClass method.  This executes once before all tests in the file and is just repeating work that is done in setUp before each test.

Reply:
There is no setUp before each test.Should I change this to @Before to do that instead?

Reply:
The setUp routine started on line 40 of the file before you modified this.